tools/wptrunner/wptrunner/executors/executormarionette.py

```diff
@@ -879,7 +942,7 @@ def _screenshot(self, protocol, url, timeout):
         if screenshot.startswith("data:image/png;base64,"):
             screenshot = screenshot.split(",", 1)[1]
 
-        return screenshot
+        return [screenshot]
```

Comment:
It seems weird for a method to return a single item in a list, and looking at all the cases where `_screenshot` has been changed that seems to be what is happening. Assuming the caller is gathering a list, could it do the stitching instead?

Also, at least in executorwebdriver.py `_screenshot` wasn't changed, and I'm not sure if that was deliberate or not?